File: 36-Extra-Challenges/110-Remove-Every-Other.py

def remove_every_other(lst):
    return [el for i, el in enumerate(lst) if i % 2 == 0]


print(remove_every_other([1,2,3,4,5]))
print(remove_every_other([5,1,2,4,1]))
print(remove_every_other([1]))

File: 36-Extra-Challenges/110-Remove-Every-Other.js

const removeEveryOther = (arr) => {
    return arr.filter((el, i) => i % 2 === 0)
}

console.log(removeEveryOther([1,2,3,4,5]))
console.log(removeEveryOther([5,1,2,4,1]))
console.log(removeEveryOther([1]))
// [1,2,3,4,5]))
// [5,1,2,4,1]))
// [1]))